pkg/securitypolicy/securitypolicy_test.go

```diff
@@ -1128,6 +1135,11 @@ func newDataGenerator(rng *rand.Rand) *dataGenerator {
 		mountTargets: map[string]struct{}{},
 		containerIDs: map[string]struct{}{},
 		sandboxIDs:   map[string]struct{}{},
+		enforcementPoints: map[string]struct{}{
+			"mount_device":     {},
+			"mount_overlay":    {},
+			"create_container": {},
+		},
```

Comment:
we would like to have this list of enforcement points that exist and shouldn't be generated from our unique enforcement point generator to not have to be updated in conjunction with the list used in GCS, however, that is currently very difficult. 

Once the build tags go away and we have only rego, it should be straightforward; as such, we have created an item on our "address later" list to make the change after we are no longer using build tags to select an enforcer.